FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -39,9 +40,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             showWelcome()
         }
 
+        oneSignalStartup(launchOptions: launchOptions)
+
         return true
     }
 
+    func oneSignalStartup(launchOptions: [UIApplicationLaunchOptionsKey: Any]?) {
+        let onesignalInitSettings = [kOSSettingsKeyAutoPrompt: false]
+
+        OneSignal.initWithLaunchOptions(launchOptions,
+                                        appId: "5fd4ca41-9f6c-4149-a312-ae3e71b35c0e",
```

Comment:
Would be nice to tuck this inside a plist so we could change it by a configuration change (by scheme perhaps).  Then we could access it with `AppConfiguration.current.setting(for: "OneSignalAppId")`.  What do you think?

Reply:
Put this in build settings, then referenced it in Info.plist so it can be accessed through `Bundle.main.infoDictionary`